components/EventDetail.js

```diff
@@ -46,6 +60,8 @@ export default function EventDetail({ event }) {
         <StyledEventInfo>{event.organizer}</StyledEventInfo>
       </StyledEventInfoContainer>
       <StyledDescription>{event.description}</StyledDescription>
+
+      <button onClick={() => handleDelete()}>Delete</button>
```

Comment:
You can skip the anonymous arrow function since you are not passing any param to `handleDelete`, so just:

```js
 <button onClick={handleDelete}>
```